.ruff.toml

```diff
@@ -0,0 +1,23 @@
+fix = true
+
+[lint]
+select = [
+  "C4",     # flake8-comprehensions
+  "E",      # pycodestyle errors
+  "F",      # pyflakes errors
+  "I",      # isort
+  "ISC",    # flake8-implicit-str-concat
+  "LOG",    # flake8-logging
+  "PGH",    # pygrep-hooks
+  "RUF100", # unused noqa (yesqa)
+  "UP",     # pyupgrade
+  "W",      # pycodestyle warnings
+  "YTT",    # flake8-2020
+]
+ignore = [
+  "E203",   # Whitespace before ':'
+  "E221",   # Multiple spaces before operator
+  "E226",   # Missing whitespace around arithmetic operator
+  "E241",   # Multiple spaces after ','
+  "E501",   # Line too long
```

Comment:
Why are these ignored?

Reply:
E2xxx because they're some common ones that are incompatible with Black. I copied these over from another project along with the `select` list. We could remove them here until adding Black if you like? They don't flag anything right now.

E501 because there's lots of long lines. We can remove this when we add Black formatting.